core/templates/dev/head/css/oppia.css

```diff
@@ -2182,11 +2182,15 @@ pre.oppia-pre-wrapped-text {
 */
 .oppia-rte-editor > p,
 .form-control.oppia-rte-content > div > p {
-  line-height: 28px;
+  line-height: 1.5;
   margin-bottom: 18px;
   margin-top: 18px;
 }
 
+.multiple-choice-content-container > span > p {
```

Comment:
This seems incorrect. The spacing choices should affect RTE-related classes and nothing else -- RTE boxes in the multiple-choice editor should be treated the same way as RTE boxes in other places.